config/lobby/db/004_audit_bans

```diff
@@ -2,7 +2,7 @@ start transaction;
 
 alter table banned_macs
   add column mod_username varchar(40) not null default '__unknown__',
-  add column mod_ip varchar(45) not null default '0.0.0.0',
+  add column mod_ip inet not null default '0.0.0.0'::inet,
```

Comment:
@ssoloff what are your thoughts on dropping the `default` from these columns? Normalizing `null` to `0.0.0.0` I do not think is best:
- It's a magic value to be remembered
- In DB context 'no value' provided is generally represented as null.
- We lose some awareness if bad data is being inserted to DB.
- I suspect it's better to fail in cases where not all data is inserted rather than allow a developer writing an insert to forget or hardcode a null for IP rather than being forced to wire it properly. I'm thinking Yoda principle on this one: “There is no try. Do."

Reply:
@DanVanAtta If you look a bit further down the script, you'll see the defaults are immediately dropped.  During the discussion in #2828, it was deemed better to ensure the new columns are always `not null` rather than allowing records that existed prior to this change to have `null` values for these columns, while new records would use `not null` values.

Bottom line, all new bans/mutes are required to provide `not null` values for these columns.  Existing bans/mutes will have default values, but these records will eventually roll off the table when they expire (unless, of course, they are Forever bans/mutes).